include/RAJA/policy/PolicyBase.hpp

```diff
@@ -69,16 +69,26 @@ enum class Pattern {
   taskgraph,
 };
 
+enum class Platform {
+  undefined = 0,
+  host = 1,
+  cuda = 2,
+  omp_target = 4
+};
+
 struct PolicyBase {
 };
 
 template <Policy P = Policy::undefined,
           Launch L = Launch::undefined,
-          Pattern Pat = Pattern::undefined>
+          Pattern Pat = Pattern::undefined,
+          Platform Plat = Platform::host
```

Comment:
It is possible that the OpenMP policy may at some point support device vs host selection at runtime, but if it does it will still be a function of the policy.